interceptor/interceptor.go

```diff
@@ -286,3 +410,36 @@ func fetch(this js.Value, args []js.Value) interface{} {
 	promise := promiseConstructor.New(js.FuncOf(promise_logic))
 	return promise
 }
+
+func getStatic(this js.Value, args []js.Value) interface{} {
+	url := args[0].String()
+
+	return js.Global().Get("Promise").New(js.FuncOf(func(this js.Value, args []js.Value) interface{} {
+		resolve := args[0]
+
+		go func() {
+			resp := L8Client.Do(
+				url, utils.NewRequest("GET", make(map[string]string), nil),
+				userSymmetricKey, true, UpJWT, UUID)
+
+			// convert response body to js arraybuffer
+			jsBody := js.Global().Get("Uint8Array").New(len(resp.Body))
+			js.CopyBytesToJS(jsBody, resp.Body)
+
+			// create a map of the response headers
+			resHeaders := js.Global().Get("Headers").New()
+			for k, v := range resp.Headers {
+				resHeaders.Call("append", js.ValueOf(k), js.ValueOf(v))
+			}
+
+			blob := js.Global().Get("Blob").New([]interface{}{jsBody}, js.ValueOf(map[string]interface{}{
```

Comment:
I can read lines 435 - 438. I can understand line 435 - 438. But I could not write it myself. Beautiful.